src/features/collapsible-sidebar/CollapsibleSidebar.scss

```diff
@@ -143,6 +143,11 @@ $left-sidebar-z-index: 40 !default;
     .Body.is-responsive-web {
         .bdl-CollapsibleSidebar-wrapper {
             width: 0;
+            margin-top: 60px;
```

Comment:
Encapsulation :) We don't want CollapsibleSidebar to be concerned with its external layout. We should apply the margin in EUA for sure.